docs/nitpick-exceptions

```diff
@@ -22,6 +22,7 @@ py:class loguru._logger.Logger
 # Type aliases don't play nicely with sphinx when you import them
 # elsewhere.  Works fine for static type checker though.  I think this
 # is because sphinx does runtime checks.
+py:class _ParameterType
```

Comment:
If you remove the `_ParameterType` as I suggested, make sure to remove this from here.